mitosheet/css/mito.css

```diff
@@ -2,6 +2,22 @@
     --taskpane-width: 430px;
 }
 
+/* 
+    The following two css rules are used to style the Create new Mitosheet button in the notebook toolbar. 
+    The pure css selectors are used for the JLab button, and the id selection is used for the notebook 
+    button, since we're not able to set the class name of our button in notebooks. 
+*/
+:root .toolbar-mito-button-class, #create_mitosheet {
```

Comment:
Why is there a root selector here? Is this necessary?

Reply:
Can we name the id `create-mitosheet-button` and the class `create-mitosheet-button-class` for symmetry?